options/list-styles.js

```diff
@@ -92,6 +92,7 @@ const FullStyleList = [
     { file: "base16/eva",                           name: "Eva" },
     { file: "base16/eva-dim",                       name: "Eva Dim" },
     { file: "far",                                  name: "Far" },
+    { file: "felipec",                              name: "Felipe Contreras" },
```

Comment:
The CSS file has:
```
 * Theme: FelipeC
 * Author: (c) 2021 Felipe Contreras <felipe.contreras@gmail.com>
```
Highlight.js also describes it as `Felipec` ([on the demo page](https://highlightjs.org/static/demo/), apparently they forgot it in the release notes??). Let's use `FelipeC` as in the file.

Reply:
Sure. Patch updated.